data/commodities.txt

```diff
@@ -867,19 +1007,32 @@ trade
 		"weaponized chickens"
 		"weapons-grade uranium"
 	commodity "void sprite parts"
+		"dismembered void sprites"
 		"void sprite parts"
 		"void sprite limbs"
-		"dismembered void sprites"
 	commodity "Military"
 		"ammunition"
 		"armor plating"
+		"armored cars"
+		"artillery"
 		"atmospheric craft"
 		"battlefield drones"
+		"bombers"
```

Comment:
Bombers is kind of vague and doesn't really specify whether it's a bomber that conducts atmospheric or orbital bombardment. You could maybe replace this with more specific types, although it's fine as-is.

Reply:
I’ll take it out.